services/cloudapi/lanSvc/lan.go

```diff
@@ -0,0 +1,22 @@
+package lanSvc
+
+import (
+	"context"
+	"fmt"
+	ionoscloud "github.com/ionos-cloud/sdk-go/v6"
+)
+
+// Currently, this is not a service per se, but in the future, when the LAN service will be created,
+// this function can be included in the service. Right now, it is just an utility function in other
```

Comment:
`in other` to `in order`

Reply:
@cristiGuranIonos done in 7a35821c28be88119348db39cb88c96db7c113b9